xml/schema.xsd

```diff
@@ -36,7 +36,7 @@
     <xsd:complexContent>
       <xsd:extension base="varField">
         <xsd:sequence>
-          <xsd:element name="value" type="xsd:string" />
+          <xsd:element name="value" type="xsd:string" minOccurs="1" maxOccurs="1"/>
     		  <xsd:element name="pattern" type="xsd:string" minOccurs="0" maxOccurs="1"/>

```

Comment:
Unecessary tabbing here, this doesn't seem to be from you, but can you change it for this PR?